pixi.toml

```diff
@@ -17,7 +17,7 @@ matplotlib = "*"
 meson = "1.3.0"
 networkx = "*"
 ninja = "*"
-numpy = "*"
+numpy = "<2.0.0"    # modflowapi has a resriction on the numpy version
```

Comment:
we need to release a new modflowapi supporting numpy2, thanks for the reminder &mdash; then this can be unpinned